dev-tools/mage/check.go

```diff
@@ -83,12 +82,19 @@ func CheckLicenseHeaders() error {
 // CheckLinksInFileAreLive checks if all links in a file are live.
 func CheckLinksInFileAreLive(filename string) func() error {
 	return func() error {
+
+		// Skip for now until we can make this work without exceeding the GitHub rate limit
```

Comment:
Add the issue URL here so anyone wandering into this comment can read more for context?
```suggestion
		// Skip for now until we can make this work without exceeding the GitHub rate limit
		// Related: https://github.com/elastic/elastic-agent/issues/8064
```